doc/overview/scopes.md

```diff
@@ -3,10 +3,12 @@ Scopes
 
 ### Introduction to scopes
 
-In Visual Studio there are three "scopes" of context related to project
-systems:
+A "scope" is a collection of MEF parts that are tied to the lifecycle of a specific object, or "context". When that object or context is torn down, so are the MEF parts that live in that scope. Scopes themselves can be nested so that parts that live in child scopes can "inherit" or import exports from parent scopes, but not vice versa.
 
-- `IVsSolution`, or the global context. There is exactly one of these in the 
+In Visual Studio, there is one global default "scope" of context and three related to project systems:
+
+- __VS Default Container__. This is the default global scope, where parts in Visual Studio live by default. There is exactly on of these in the process.
+- `IVsSolution`, or the solution context. There is exactly one of these in the 
```

Comment:
I think we should avoid referring to the `ProjectService` scope as the solution or `IVsSolution` scope. `ProjectService` has the same lifetime as the VS default container, whereas association with a solution suggests it has solution data and its lifetime gets recycled between loaded solutions, which is not correct.